### PR TITLE
Revenant Gamemode

### PR DESCRIPTION
**Pull request by Geevies**
* Added a new wave defense gamemode, Revenants, wherein the station crew have to defend themselves from an endless wave of humanoid bluespace creatures.

Ported the icon from Baystation, it's their blueforged, but I tweaked the mob's alpha values a tad. https://github.com/Baystation12/Baystation12/pull/21669

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Review by Alberyk — DISMISSED** [review]
The shadow people/grue sprites don't really match the intent/lore of the antagonist. If you want this to be a gamemode with some kind of extradimensional invader, you will have to get more fitting sprites instead of reusing poorly what exists.

**Comment by TheDanseMacabre** [comment]
Yonnimer ran a wishgranter event recently with a bluespace creature that could be useful here.

File: code/modules/mob/living/carbon/human/human_powers.dm [hunk 1]
```diff
@@ -583,6 +583,15 @@ mob/living/carbon/human/proc/change_monitor()
 	A.throw_at(target, 10, 30, usr)
 	msg_admin_attack("[key_name_admin(src)] launched a quill at [key_name_admin(target)] (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[src.x];Y=[src.y];Z=[src.z]'>JMP</a>)",ckey=key_name(src),ckey_target=key_name(target))
 
+/mob/living/carbon/human/proc/dissolve()
+	set name = "Dissolve Self"
+	set desc = "Dissolve yourself in order to escape permanent imprisonment."
+	set category = "Abilities"
+
+	if(alert(usr, "This ability to kill you, are you sure you want to do this?", "Dissolve Self", "Yes", "No") == "No")
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/human_powers.dm:591]: `This ability to kill you` wut

**Comment by Geevies** [comment]
!review

**Review by skull132 — APPROVED** [review]

**Comment by Geevies** [comment]
Tested it again for good measure, still works as intended.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**